src/frontend/src/instance/standalone.rs

```diff
@@ -144,6 +145,11 @@ impl TableMetadataAllocator for StandaloneTableMetadataCreator {
             })
             .collect::<Vec<_>>();
 
-        Ok((table_id, region_routes))
+        // There're no wal options involved in standalone mode currently.
```

Comment:
I think standalone can use `kafka` as wal too. For example, I want to use a docker container to run GreptimeDB, and store the data in s3, and keep wal in Kafka to make sure all the data will not be lost in a crash.

Reply:
Added a `WalOptionsAllocator` into `StandaloneTableMetadataAllocator`.

Reply:
But for now, the `WalOptionsAllocator` is default initialized. The initialization of the allocator involves codes in another PR. So will initialize it in the following PRs.